Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Toast/ToastCode.bind

```diff
@@ -18,7 +12,6 @@ public static ToastContent GenerateToastContent()
                                          ("240", "4 hours"),        
                                          ("1440", "1 day"))        
         .AddButton(new ToastButtonSnooze() { SelectionBoxId = "snoozeTime" })        
-        .AddButton(new ToastButtonDismiss());        
-
-    return builder.Content;        
-}        
\ No newline at end of file
+        .AddButton(new ToastButtonDismiss());
```

Comment:
I think this semicolon is wrong here, no? Have you executed this on the sample app?